### ISSUE TITLE
GoTest shows failure on passing test

### ISSUE THREAD

**Issue by lancekrogers** [ray-x/go.nvim#379]
Running :GoTest on a test that has no failures is showing the below error 
<img width="755" alt="Screenshot 2023-09-21 at 12 55 05 PM" src="https://github.com/ray-x/go.nvim/assets/12162551/48fb02db-1641-40e3-a250-6db5897e186d">

The test with not modification passes when ```go test ./...``` is run direction from the command line

<img width="642" alt="Screenshot 2023-09-21 at 1 21 06 PM" src="https://github.com/ray-x/go.nvim/assets/12162551/41d1f03b-1ff2-45ab-a5d8-6e9cf1577ed6">

I am experiencing this with commit 5e9e083 but don't know when it started.  I first noticed this issue yesterday after updating lunarvim.

**Comment by ray-x** [comment]
Is the folder in first screen shot looks correct? what is the current working folder in nvim when you run the tests?

**Comment by lancekrogers** [comment]
Yes the folder is correct, when I run ```:lua print(vim.loop.cwd())``` the output is ```~/{user}/Dev/learn-go-with-tests```

**Comment by eduardosanzb** [comment]
I was having same issues; but I ensure that I was running inside the same directory as my golang project (working in a monorepo)

Would be a way to change the root for `GoTest`?
